pkg/config/remote/uptane/client.go

```diff
@@ -360,3 +362,33 @@ func (c *Client) verifyUptane() error {
 	}
 	return nil
 }
+
+func configMetasUpdateSummary(metas *pbgo.ConfigMetas) string {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏It'd be great if we could test it

Reply:
Fair. I validated it manually modifying a unit test to actually print the error but a test could be added confirming the error string. (although I don't love tests on the "string" representation of the error as its brittle, probably the best path is to make a validation failure like this a typed error with fields for all these values, and test that the error is of the right type with the right values when type cast.) Given the freeze I'll merge this now and if I can make that refactor today I will, otherwise will create a card for it.